lib/travis/build/script/go.rb

```diff
@@ -110,10 +110,10 @@ def uses_make
           
           # see https://golang.org/doc/go1.6#go_command
           def uses_15_vendoring
-            if (go_version == 'go1' || comparable_go_version < Gem::Version.new('1.5'))
+            if (go_version == 'go1' || (go_version != 'tip' && comparable_go_version < Gem::Version.new('1.5')))
```

Comment:
Instead of `go_version != 'tip'`, how about having a method with more descriptive name, say `tip?` or `version_is_tip?`?